lib/caseStatus.ts

```diff
@@ -34,6 +34,14 @@ export const addCaseStatus = async (
   });
 };
 
+export const updateCaseStatus = async (
+  params: Record<string, unknown>
+): Promise<void> => {
+  await axios.post(`${ENDPOINT_API}/residents/case-statuses`, params, {
```

Comment:
This needs to follow the new endpoint defined in SwaggerHub

/case-statuses/{id}/answers